lib/kafee/consumer/brod_adapter.ex

```diff
@@ -24,6 +24,20 @@ defmodule Kafee.Consumer.BrodAdapter do
                       The retry backoff time in milliseconds.
                       """,
                       type: :integer
+                    ],
+                    offset_reset_policy: [
+                      default: :reset_by_subscriber,
```

Comment:
```suggestion
                      default: :reset_to_earliest,
```

We should just likely make this the default unless we add in offset tracking 🫠

Reply:
I can make it in a separate PR to separate the breaking change.